### PR TITLE
{bio}[foss/2019b] MRtrix v3.0_RC4 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2454.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a49be3f81661ee03f784b98f8f0465c0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3124.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/652cb87896db351cd8fa5545399a9e0c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso-2 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/c550bff0dc2ef55057cad2edaa974a93 for a full test report.

File: easybuild/easyconfigs/m/MRtrix/MRtrix-3.0_RC4-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+source_urls = ['https://github.com/MRtrix3/mrtrix3/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['4f246e4cc6ddd5d2480d368342bcbeb9c5d21c04a3a4b5e6e2adb8c036611426']
+
+builddependencies = [
+    ('Eigen', '3.3.7', '', True),
+    ('pkg-config', '0.29.2'),
+]
+dependencies = [
+    ('zlib', '1.2.11'),
```
Comment (smoors) [inline easybuild/easyconfigs/m/MRtrix/MRtrix-3.0_RC4-foss-2019b-Python-3.7.4.eb]: @boegel `zlib` is already a dep of foss, so can it be removed?

File: easybuild/easyconfigs/m/MRtrix/MRtrix-3.0_RC4-foss-2019b-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    ('Python', '3.7.4'),
+    ('Mesa', '19.1.7'),
+    ('Qt5', '5.13.1'),
+    ('LibTIFF', '4.0.10'),
+    ('FFTW', '3.3.8'),
```
Comment (smoors) [inline easybuild/easyconfigs/m/MRtrix/MRtrix-3.0_RC4-foss-2019b-Python-3.7.4.eb]: @boegel same here

File: easybuild/easyconfigs/m/MRtrix/MRtrix-3.0_RC4-foss-2019b-Python-3.7.4.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/m/MRtrix/MRtrix-3.0_RC4-foss-2019b-Python-3.7.4.eb]: @smoors `zlib` is only an indirect dep of `foss`, so it's better to leave it in as an explicit dependency imho...

File: easybuild/easyconfigs/m/MRtrix/MRtrix-3.0_RC4-foss-2019b-Python-3.7.4.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/m/MRtrix/MRtrix-3.0_RC4-foss-2019b-Python-3.7.4.eb]: dropping this makes sense I guess, but then it's easy to overlook that `FFTW` is required when switching to `intel` as toolchain...

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node375.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/ebe52bafd09ad7dbfa80084009eb6b44 for a full test report.

**Comment by smoors** [comment]
(ran out of memory)

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node373.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/537398da8ad3c8c8788a187b8ce1fb7c for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node374.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/ea2f05b698596eefe7270ee5a999ea68 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Pull request merged**